src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String lineBreak = "---------------------------------------------";
+        String line = "";
+        Task[] list = new Task[100];
```

Comment:
Code Standards: Perhaps this should be plural to represent a collection of objects?

```suggestion
        int MAX_TASK = 100;
        Task[] lists = new Task[MAX_TASK];
```